source/components/patternfly/tests-src/_includes/widgets/list-view-variation-1.html

```diff
@@ -0,0 +1,121 @@
+<div class="list-group list-view-pf">
```

Comment:
We don't want to check this file into the widgets area. The widgets directory is only a soft link to the version of patternfly installed by bower.

This file should only be pushed to the patternfly repo.